### DISCUSSION TITLE
Instructions to Setup next using env variables and docker

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2998] [Archive]
Hi.

I want to try next and I can't find instructions to set up automatically DB via the cmd line. 

Do we use the same Env variables for that: https://docs.directus.io/docker/environment.html#database? 

What if Postgres has some extra connection options like SSL mode, how do we set that? 

Thx.

**Comment by benhaynes** [comment]
Is this what you're looking for?

https://github.com/directus/next/blob/main/docs/reference/environment-variables.md

**Reply by botzill** [reply]
Thx @benhaynes, that's what I'm looking for.

**Reply by botzill** [reply]
@benhaynes Is there any place I can set up the default admin email, password?

**Reply by botzill** [reply]
OK, found
DIRECTUS_ADMIN_EMAIL
DIRECTUS_ADMIN_PASSWORD

right?

**Reply by benhaynes** [reply]
Hmm, I don't think you can set the initial admin from within the `env` file, I think it needs to be set via the CLI or directly in the database. Right @rijkvanzanten ?

**Reply by rijkvanzanten** [reply]
@benhaynes The Docker setup allows for it, otherwise there's no way to login after first initialization. @WoLfulus what's the env var in docker for that?
